app/controllers/UpscanController.scala

```diff
@@ -206,7 +206,7 @@ class UpscanController @Inject()(repository: UploadJourneyRepository,
                     .addingToSession(SessionKeys.failureMessageFromUpscan -> failureReason))
                 }
               } else {
-                Future(Redirect(controllers.routes.OtherReasonController.onPageLoadForUploadComplete()))
+                Future(Redirect(controllers.routes.OtherReasonController.onPageLoadForAnotherFileUpload(NormalMode)))
```

Comment:
We always go to the UploadComplete page i.e. you have uploaded 'X' files page, it makes sense to revert this